_layouts/api.html

```diff
@@ -17,8 +17,61 @@ <h2 class="ui inverted header">
 
 </section>
 
-<div id="description" class="ui text container markdown-body">
-  {{ content }}
+<div id="description" class="ui text container ">
+  <div class="ui segment markdown-body">
+    <div class="ui right dividing rail">
+      <div class="ui inverted segment">
+        <div class="ui inverted relaxed divided list">
+          <div class="item">
+            <div class="content">
+              <div class="header">Conditions d'utilisation</div>
+              <div class="description">
+                {{ page.contract }}
+              </div>
+            </div>
+          </div>
+          {% if page.restriction %}
+          <div class="item">
+            <div class="content">
+              <div class="header">Contraintes</div>
+              <div class="description">
+                {{ page.restriction }}
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          <div class="item">
+            <div class="content">
+              <div class="header">Clients cibles</div>
+              <div class="description">
+              {% for client in page.clients %}
+                {{ client }}
+                {% if forloop.last == false %}
+                &amp;
+                {% endif %}
+              {% endfor %}
+              </div>
+            </div>
+          </div>
+          <div class="item">
+            <div class="content">
+              <div class="header">Sources et partenaires</div>
+              <div class="description">
+              {% for partner in page.partners %}
+                {{ partner }}
+                {% if forloop.last == false %}
+                &amp;
```

Comment:
Sure you don't want to simply [`{{ page.partners | join: '&amp;' }}`](https://docs.shopify.com/themes/liquid/filters/array-filters#join)? :wink: